Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/UI/RecordingServiceVisual.cs

```diff
@@ -93,17 +93,15 @@ protected void Awake()
                 Debug.LogError("Error: Recording service not set for RecordingServiceVisual");
                 return;
             }
-
-            _recordButton.onClick.AddListener(OnRecordClick);
-            _previewButton.onClick.AddListener(OnPreviewClick);
         }
 
         protected void Update()
         {
             if (_recordingService != null &&
                 _previewButton != null)
             {
-                _previewButton.gameObject?.SetActive(_recordingService.IsRecordingAvailable());
+                bool showPreviewButton = _recordingService.IsRecordingAvailable() && _recordButton.gameObject.activeSelf;
+                _previewButton.gameObject?.SetActive(showPreviewButton);
```

Comment:
>gameObject?. [](start = 31, length = 12)

Because Unity has an override for Equals and operator== to treat a non-null CLR object that represents a destroyed GameObject/Component as 'null', the "?." and "??" operators don't work like you'd expect with Component or GameObject classes (because they explicitly call ReferenceEquals(o, null) and don't call through overrides of Equals or operator==).  You instead have to have a traditional null check for GameObjects or Component.